### PR TITLE
remove h1 on navbar desktop / mobile

### PR DESCRIPTION
**Pull request by NyKo24**

### CODE REVIEW

File: front/style/mixins/_text.scss [hunk 1]
```diff
@@ -513,7 +513,7 @@ $text--size-extra-large: 48px;
             font-size: $text--size-default;
             letter-spacing: 0;
 
-            h1 {
+            .text--logo {
```
Comment (marineb) [inline front/style/mixins/_text.scss]: I think you could completely remove this class, no ? Since there's already a `text--logo` class that exists.

Comment (NyKo24) [reply front/style/mixins/_text.scss]: ah ? Where ?

Comment (marineb) [reply front/style/mixins/_text.scss]: Higher up in that same file. You have `--logo`. That's attached to `.text`.

Comment (NyKo24) [reply front/style/mixins/_text.scss]: ✅

**Pull request merged**